lib/shoulda/matchers/action_controller/flash_store.rb

```diff
@@ -34,10 +34,8 @@ def has_value?(expected_value)
             expected_value === actual_value
           end
         end
-
-        def empty?
-          flash.empty?
-        end
+        
```

Comment:
It looks like Rubocop is detecting that there is extra whitespace here. Maybe there is an extra space? Can you check this line? Sorry